data/gegno/gegno intro missions.txt

```diff
@@ -427,7 +427,7 @@ mission "Giaru Gegno: Revisit"
 					goto "trouble"
 			`	"Indeed yes, strange coincidences do curiously happen. Many coincidences are as mere fleeting moments of subtle peculiarity." The Quarg then displays a soft grimace. "Sometimes, one may be grand and powerful, in addition to having potent consequences. The Gegno have learned this a great number of times.`
 			label "trouble"
-			`	"Quite a brave one you indeed are. Most would think to not leap into the daunting face of uncertainty as you have." The Quarg momentarily meditates on its next thought. "Though, as the Gegno have throughout their history, some may make monumentous jumps undeterred by any amount of hesitance."`
+			`	"Quite a brave one you indeed are. Most would think to not leap into the daunting face of uncertainty as you have." The Quarg momentarily meditates on its next thought. "Though, as the Gegno have throughout their history, some may make monumental jumps undeterred by any amount of hesitance."`
```

Comment:
```suggestion
			`	"Quite a brave one you indeed are. Most would think to not leap into the daunting face of uncertainty as you have." The Quarg momentarily meditates on its next thought. "Though, as the Gegno have throughout their history, some may make momentous jumps undeterred by any amount of hesitance."`
```

I was going for momentous here:

mo·men·tous
/mōˈmen(t)əs/
adjective
(of a decision, event, or change) of great importance or significance, especially in its bearing on the future.

Seems I jumbled up the two words, yes.